content/en/blog/_posts/2023-04-14-improving-support-go-updates.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: blog
+title: "Improving Support: Go Updates"
```

Comment:
```suggestion
title: "Kubernetes Build Process: Compatibility With Releases Of Go"
```
?

Reply:
mmm... it's not really the build process that's changing here

Reply:
the net effect is making Kubernetes patch releases more secure, because we can now use up-to-date go versions... 

brainstorming a little... maybe something combining "Kubernetes Patch Releases", "Go", and some combination of "secure / supported / current" verbiage

Reply:
maybe `Kubernetes Patch Releases: Staying On Supported Go Versions`?